usaspending_api/download/filestreaming/download_generation.py

```diff
@@ -531,33 +552,46 @@ def _top_level_split(sql, splitter):
 
 def execute_psql(temp_sql_file_path, source_path, download_job):
     """Executes a single PSQL command within its own Subprocess"""
-    try:
-        log_time = time.perf_counter()
-        temp_env = os.environ.copy()
-        if download_job and not download_job.monthly_download:
-            # Since terminating the process isn't guaranteed to end the DB statement, add timeout to client connection
-            temp_env["PGOPTIONS"] = f"--statement-timeout={settings.DOWNLOAD_DB_TIMEOUT_IN_HOURS}h"
-
-        cat_command = subprocess.Popen(["cat", temp_sql_file_path], stdout=subprocess.PIPE)
-        subprocess.check_output(
-            ["psql", "-q", "-o", source_path, retrieve_db_string(), "-v", "ON_ERROR_STOP=1"],
-            stdin=cat_command.stdout,
-            stderr=subprocess.STDOUT,
-            env=temp_env,
-        )
+    # TODO: The datadog trace says the SQL is not parseable. Could be the prepended COPY. Get raw sql select
+    sql = subprocess.check_output(["cat", temp_sql_file_path]).decode()
```

Comment:
I realize this line was moved, but if all it needs is the file contents using `pathlb` would be cleaner.

```python
sql = Path(temp_sql_file_path).read_text()
```

Reply:
👍 Will use